packages/editor/src/components/copy-handler/index.js

```diff
@@ -26,18 +26,18 @@ class CopyHandler extends Component {
 	}
 
 	onCopy( event ) {
-		const { multiSelectedBlocks, selectedBlock } = this.props;
+		const { selectedBlockClientIds, getBlocks } = this.props;
```

Comment:
```suggestion
		const { selectedBlockClientIds = [], getBlocks } = this.props;
```
would it be possible to use to simplify next conditional check?

Reply:
Actually it's an array in all cases, I'll just remove the first check.